src/MaxwellSolvers/Maxwell.h

```diff
@@ -15,32 +15,31 @@
 
 namespace ippl {
 
-    template <typename EMField, typename SourceField>
+    template <typename EMField, typename FourField>
```

Comment:
merge with master to have the new class

Reply:
this still has merge conflicts with the master apparently, please fix

Reply:
Merged with master.